cluster.tf

```diff
@@ -20,7 +20,7 @@ module "cluster" {
 
   eks_managed_node_groups = local.cluster.install ? {
     default = {
-      instance_types                        = ["t3.medium"]
+      instance_types                        = local.cluster["default"].instance_types
```

Comment:
This should be `local.cluster.managed_node_groups.default.instance_types`, but we should probably also just use the full list passed in?